Gopkg.lock

```diff
@@ -432,28 +433,9 @@
   ]
   revision = "39a7bf85c140f972372c2a0d1ee40adbf0c8bfe1"
 
-[[projects]]
```

Comment:
I think this difference stems from you doing things on a copy created from your repo, whereas I did this on `k8s.io/sample-controller` and then added my fork as a new remote (so this isn't needed because it's us!).  If we want to call this something other than `k8s.io/sample-controller`, then I think we need to rewrite references so that we don't have a dependency on it.

Reply:
Just to confirm I understand: my process was

```sh
(fork)
dep init
```

and your process was
```sh
dep init
(fork)
```
> If we want to call this something other than k8s.io/sample-controller, then I think we need to rewrite references so that we don't have a dependency on it.

:+1: I expect `dep` will gracefully detect future package renames.

Reply:
I never called `dep init`, since I think that's one-time setup.

What I did was:
```shell
go get k8s.io/sample-controller
cd $GOPATH/src/k8s.io/sample-controller
git remote add mattmoor <my fork>
./hack/update-deps.sh
```

So my point was that I'm working within a directory named `k8s.io/sample-controller` on my `GOPATH`.

Reply:
> So my point was that I'm working within a directory named `k8s.io/sample-controller` on my `GOPATH`.

Do we expect this fork to ultimately be merged back into the parent? If not I think it makes sense to choose a new prefix.

Reply:
I doubt it.  I think this PR makes it *slightly* more correct (since it isn't depending on vendored code, but it's own copy), but I think a subsequent commit that `s@k8s.io/sample-controller@foo.bar/baz@g` would be fine.

Reply:
Just tested it; changing `go_prefix` and rerunning `hack/update-deps.sh` was enough to update everything.